### ISSUE TITLE
[Bug]: same language code("en") for all users

### ISSUE THREAD

**Issue by zion3w** [OneSignal/OneSignal-Android-SDK#1843]
### What happened?

start using latest sdk, and there is always "en" language code for all users. had test on a device with "ru" lang, and have "en" in onesignal dashboard.

### Steps to reproduce?

```Markdown
1. install on a device with not "en" language and check dashboard
```

### What did you expect to happen?

to see real device language tag in dashboard) how I'm going to send messages on a different language users if it's "en" only ?)

### OneSignal Android SDK version

5.0.0

### Android version

13, 12, 11, 10, 9, 8, 7.1, 7.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by whyamsx** [comment]
https://github.com/OneSignal/OneSignal-Android-SDK/blob/main/OneSignalSDK/onesignal/src/main/res/values/strings.xml

Also this messages need translate to method **promptForPushNotificationsWithUserResponse**
I want set another language to this messages, but not have possibility

**Comment by jennantilla** [comment]
Hi @zion3w thanks for reaching out! We're working on a fix for this and will let you know when it's released!

**Comment by jennantilla** [comment]
@zion3w thanks for your patience! This fix has been included in our latest [release 5.0.2](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/5.0.2). Let us know if you have any other questions or concerns.
